src/main/java/org/gbif/ipt/action/manage/AutoPublishAction.java

```diff
@@ -13,6 +13,7 @@
  */
 package org.gbif.ipt.action.manage;
 
+import java.util.TimeZone;
```

Comment:
We usually group imports (sth like GBIF ones,  java.* ones, etc together.). In some projects it's performed automatically, but not in the IPT. It's not mandatory though.